notes.md

```diff
@@ -44,11 +44,6 @@ The default value of an Element is assumed when not present in the data stream.
 
 Digital Rights Management. See [Encryption]({{site.baseurl}}/notes.html#Encryption).
 
-# EBML Class
-
-A larger EBML Class typically means the Element has a lower probability/importance. A larger Class-ID can be used as a synch word in case the file is damaged. Elements that are used frequently, but do not need to act as a synch word, SHOULD have a small Class-ID.
```

Comment:
I found this in the ebml spec
> To reduce the risk of false positives while parsing EBML Streams, the IDs of the Root Element and Top-Level Elements SHOULD be at least 4 octets in length. Element IDs defined for use at Level 0 or Level 1 MAY use shorter octet lengths to facilitate padding and optimize edits to EBML Documents; for instance, the EBML Void Element uses an Element ID with a one octet length to allow its usage in more writing and editing scenarios.

which is similar enough to the above statement on Class IDs, so I'm ok with the cut.